### PR TITLE
Updated the display names for a few connectors.

### PR DESCRIPTION
**Pull request by DerekRushton**

### CODE REVIEW

File: connectors/rhacs/configurations/config.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "connection": {
         "type": {
             "type": "connectorType",
-            "displayName": "Red Hat Advanced Cluster Security for Kubernetes",
+            "displayName": "Red Hat Advanced Cluster Security for Kubernetes (StackRox)",
```
Comment (mdazam1942) [inline connectors/rhacs/configurations/config.json:5]: revert this change. we already had the correct name

**Comment by DerekRushton** [comment]
Opened a new PR.
